spec/order_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/order.rb'
+
+describe Order do
+	it 'has somewhere to store dishes chosen by user' do
+		expect(subject.basket).to eq([])
+	end
+
+	it 'is able to add a dish to the basket' do
+		subject.add(2)
```

Comment:
Again, I did something similar here but going forward I think we could both probably use mocking for a test like this so that if the menu changes the testing of the add method will still pass.